llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -592,8 +663,223 @@ static Error applyAddends(MCCASReader &Reader,
   return Error::success();
 }
 
-Expected<uint64_t> DebugInfoSectionRef::materialize(MCCASReader &Reader,
-                                                    raw_ostream *Stream) const {
+static Expected<uint64_t> getFormSize(dwarf::Form FormVal, dwarf::FormParams FP,
+                                      ArrayRef<char> CUData, uint64_t CUOffset,
```

Comment:
Instead of calling `toStringRef` with every use, let's use have this method take a `StringRef`

Reply:
Good catch!